docs/dev-environments/besu/docker-compose.yaml

```diff
@@ -4,7 +4,7 @@ services:
   # node1 is first of the two-node ethereum private network based on Hyperledger Besu
   #
   node1.avalon.org:
-    image: hyperledger/besu:1.4
+    image: hyperledger/besu:1.3.9
```

Comment:
Why is Besu being downgraded to 1.3.9?  Add that in the commit message or comment here.

Reply:
That's already there in the commit message @danintel . Provided link to the filed issue as well.